steps/const.go

```diff
@@ -42,13 +42,17 @@ const (
 	CacheRestoreCocoapodsVersion = "1"
 	CacheRestoreCarthageID       = "restore-carthage-cache"
 	CacheRestoreCarthageVersion  = "1"
+	CacheRestoreNPMID            = "restore-npm-cache"
+	CacheRestoreNPMVersion       = "1"
```

Comment:
NOTE: Do we want to update all the steps to the latest version in the default workflows (for example git-clone has a new major version)

Reply:
Good idea, but let's not do that in this PR. I'll do it in a separate PR before deploying the new `bitrise-init`